### PR TITLE
Committee templates

### PR DESCRIPTION
**Pull request by wordshaker**
## Description Of Change

Adding templates to the repository for:
- Reaching out to nomination committee members
- Reaching out to board candidates

A little bit of boy-scouting to clean up the repo ✌🏽 
- [move file which should be in meetings folders](https://github.com/dotnet-foundation/wg-membership/pull/63/commits/b68a971ff49148e1c7c7b6c921096b13b0e394d3)
- [move timelines into resources folder](https://github.com/dotnet-foundation/wg-membership/pull/63/commits/4685ab2b46e154ab980f49caeb2710f49eb6e21a)
- [move actions out of ISSUE_TEMPLATES](https://github.com/dotnet-foundation/wg-membership/pull/63/commits/c7ccc6b56d0edef733716ff6106467f5aa0e34ce)

## Purpose Of Change

To help guide future members of the working group and save some repeated work.

### CODE REVIEW

File: templates/recruiting-board/email-template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Hi _Name_
+
+I'm reaching out as I think you would be a valuable addition to the .NET Foundations Board.
+
+Part of why I think you would be great for this is _add some personal detail helping the candidate recognize why they would be great_
+
+Below is some more detail about what is entailed: 
+
+**Time Commitments**
+-
```
Comment (wordshaker) [inline templates/recruiting-board/email-template.md:10]: @BillWagner - I need your help and guidance filling this bit in

File: templates/recruiting-board/email-template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Part of why I think you would be great for this is _add some personal detail helping the candidate recognize why they would be great_
+
+Below is some more detail about what is entailed: 
+
+**Time Commitments**
+- 
+
+**Responsibilities**
+-
```
Comment (wordshaker) [inline templates/recruiting-board/email-template.md:13]: @BillWagner - I need your help and guidance filling this bit in

**Review by jguadagno — APPROVED** [review]
Awesome.  Thanks @wordshaker

File: resources/templates/recruiting-board/email-template.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Part of why I think you would be great for this is _add some personal detail helping the candidate recognize why they would be great_
+
+Below is some more detail about what is entailed: 
+
+**Time Commitments**
+- 
+
+**Responsibilities**
+-
```
Comment (BillWagner) [inline resources/templates/recruiting-board/email-template.md:13]: Here's a first draft. The linked doc has a few pages of details.

```suggestion
- Set goals for the foundation.
- Set strategic direction to achieve those goals.
- Set the yearly budget to ensure the financial wellbeing of the foundation.
- Manages the foundation’s performance against the yearly budget.
- You can learn more on the [foundation web site](https://dotnetfoundation.sharepoint.com/Shared%20Documents/Forms/AllItems.aspx?id=%2FShared%20Documents%2FMembership%20Committee%2FNomination%20Committee%20Documents%2FBoard%2Dmember%2Dresponsibilities%2Epdf&parent=%2FShared%20Documents%2FMembership%20Committee%2FNomination%20Committee%20Documents&p=true&ga=1)
```

**Review by BillWagner — APPROVED** [review]
This looks great @wordshaker 

I filled in the two sections you asked for using suggestions. If you like those, let's :shipit:

File: resources/templates/recruiting-board/email-template.md (hunk 3 above)
Comment (wordshaker) [reply resources/templates/recruiting-board/email-template.md:15]: Just so you know that link isn't public. Otherwise I am happy with this 😄

Comment (BillWagner) [reply resources/templates/recruiting-board/email-template.md:15]: I'll try to find a public link.

Comment (wordshaker) [reply resources/templates/recruiting-board/email-template.md:15]: Danke. Worst comes to worst, people can request access with the button like I just did I guess.

Comment (BillWagner) [reply resources/templates/recruiting-board/email-template.md:15]: I'll make a separate PR that makes that content public in markdown format. There's nothing confidential in it, and that will make it easier to update.

Comment (BillWagner) [reply resources/templates/recruiting-board/email-template.md:15]: I edited the link out. See #68 where I'll add a link to the new public comment once this is merged.

**Pull request merged**